azure-iot-device/tests/provisioning/test_provisioning_device_client.py

```diff
@@ -0,0 +1,14 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import pytest
+from azure.iot.device.provisioning.provisioning_device_client import ProvisioningDeviceClient
+
+
+def test_raises_exception_on_init_of_abstract_transport(mocker):
```

Comment:
I don't really believe in testing an abstract class, but I guess that's part of a larger testing philosophy discussion that's soon incoming as a result of the pipeline changes.